README.rst

```diff
@@ -0,0 +1,97 @@
+.. role:: bash(code)
+   :language: bash
+
+
+pyUmbral
+========
+
+.. image:: https://travis-ci.org/nucypher/pyUmbral.svg?branch=master
+    :target: https://travis-ci.org/nucypher/pyUmbral
+
+pyUmbral is a 100% python implementation of Umbral using OpenSSL and Cryptography.io,
```

Comment:
Actually, this isn't accurate. It uses C OpenSSL bindings w/ cryptography.io

Reply:
Right, I'll remove it.

Reply:
:heavy_check_mark: